Doppler.PushContact.Test/Controllers/DomainControllerTest.cs

```diff
@@ -193,5 +193,135 @@ public async Task Upsert_should_return_internal_server_error_when_service_throw_
+
+            // Act
+            var response = await client.SendAsync(request);
+            _output.WriteLine(response.GetHeadersAsString());
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+        }
+
+        [Fact]
+        public async Task GetPushFeatureStatus_should_response_not_found_when_domain_service_return_null()
+        {
+            // Arrange
+            var fixture = new Fixture();
+
+            var name = fixture.Create<string>();
+            Domain domain = null;
+
+            var domainServiceMock = new Mock<IDomainService>();
+            domainServiceMock.Setup(x => x.GetByNameAsync(name))
+                .ReturnsAsync(domain);
+
+            var client = _factory.WithWebHostBuilder(builder =>
+            {
+                builder.ConfigureTestServices(services =>
+                {
+                    services.AddSingleton(domainServiceMock.Object);
+                });
+
+            }).CreateClient(new WebApplicationFactoryClientOptions());
+
+            var request = new HttpRequestMessage(HttpMethod.Get, $"domains/{name}/isPushFeatureEnabled");
+
+            // Act
+            var response = await client.SendAsync(request);
+            _output.WriteLine(response.GetHeadersAsString());
+
+            // Assert
+            Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
+        }
+
+        [Theory]
+        [InlineData(true, "true")]
+        [InlineData(false, "false")]
+        public async Task GetPushFeatureStatus_should_response_push_feature_status_returned_by_domain_service(bool isPushFeatureEnabled, string expectedIsPushFeatureEnabled)
+        {
+            // Arrange
+            var fixture = new Fixture();
+
+            var name = fixture.Create<string>();
+            var domain = fixture.Create<Domain>();
+            domain.IsPushFeatureEnabled = isPushFeatureEnabled;
+
+            var domainServiceMock = new Mock<IDomainService>();
+            domainServiceMock.Setup(x => x.GetByNameAsync(name))
+                .ReturnsAsync(domain);
+
+            var client = _factory.WithWebHostBuilder(builder =>
+            {
+                builder.ConfigureTestServices(services =>
+                {
+                    services.AddSingleton(domainServiceMock.Object);
+                });
+
+            }).CreateClient(new WebApplicationFactoryClientOptions());
+
+            var request = new HttpRequestMessage(HttpMethod.Get, $"domains/{name}/isPushFeatureEnabled");
+
+            // Act
+            var response = await client.SendAsync(request);
+            _output.WriteLine(response.GetHeadersAsString());
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            var isPushFeatureEnabledResponse = await response.Content.ReadAsStringAsync();
+            Assert.Equal(expectedIsPushFeatureEnabled, isPushFeatureEnabledResponse);
```

Comment:
In my opinion, `expectedResponse` would be more semantically correct.

```suggestion
            Assert.Equal(expectedResponse, isPushFeatureEnabledResponse);
```